__tests__/api.test.js

```diff
@@ -72,6 +72,27 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "11",
+        });
```

Comment:
It looks like you've amended a previous test here? The testing should just be a new test (the one below). If the test breaks when you remove comment_count, you may need to use a more flexible matcher, as the test could be too brittle 👍